cmd/commands/server/command_server.go

```diff
@@ -121,13 +121,13 @@ func (cmd *Command) Start() (err error) {
 	return nil
 }
 
-func detectCountry(ipResolver ip.Resolver, locationDetector location.Detector) (dto_discovery.Location, error) {
+func detectCountry(ipResolver ip.Resolver, locationResolver location.Resolver) (dto_discovery.Location, error) {
```

Comment:
We can re-use country detector here as well to avoid duplicating logic behind detect country.

Reply:
done reusing